R/TaskGeneratorMoons.R

```diff
@@ -57,9 +57,9 @@ TaskGeneratorMoons = R6Class("TaskGeneratorMoons",
       x = c(runif(n1, 0, pi), runif(n2, pi, 2 * pi))
 
       data.table(
-        y = factor(rep(c("A", "B"), c(n1, n2)), levels = c("A", "B")),
-        x1 = 5  * cos(x) + rnorm(n, mean = mu, sd = sigma),
-        x2 = 10 * sin(x) + rnorm(n, mean = mu, sd = sigma)
+        y  = factor(rep(c("A", "B"), c(n1, n2)), levels = c("A", "B")),
+        x1 =              5 * cos(x) + rnorm(n, mean = mu, sd = sigma),
+        x2 =             10 * sin(x) + rnorm(n, mean = mu, sd = sigma)
```

Comment:
@lorenzwalthert This indentation looks off - is this causes because l61/62 start with numbers?

Reply:
indention or spacing after the first `=`? It was aligned before and styler does not allow complicated left alignment, so I made it right-aligned. We can also unalign if you want.

Reply:
Ah now I see - `cos` and `sin` were aligned before but complicated left alignment is not possible. 

I think right alignment here is somehow non-standard - I think I'd prefer non-matching left alignment after `=`.

```r
        x1 = 5 * cos(x) + rnorm(n, mean = mu, sd = sigma),
        x2 = 10 * sin(x) + rnorm(n, mean = mu, sd = sigma)
```